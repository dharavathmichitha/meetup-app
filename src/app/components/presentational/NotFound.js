import React from 'react';

const NotFound = () => {
  return <div className="page-not-found marginTop">The page you requested is not found... :(</div>;
};

export default NotFound;
