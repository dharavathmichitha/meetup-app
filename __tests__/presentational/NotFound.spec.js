import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../../src/app/components/presentational/NotFound';

beforeEach(() => {
  jest.resetModules();
});

describe('NotFound Component', () => {
  it('should match snapshot including default props', () => {
    const component = renderer.create(<NotFound />);

    const componentTree = component.toJSON();

    expect(componentTree).toMatchSnapshot();
  });
});
