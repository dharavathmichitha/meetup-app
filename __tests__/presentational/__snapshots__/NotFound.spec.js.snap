// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component should match snapshot including default props 1`] = `
<div
  className="page-not-found marginTop"
>
  The page you requested is not found... :(
</div>
`;
